frontend/src/app/testQueue/TestQueue.tsx

```diff
@@ -333,6 +333,7 @@ const TestQueue: React.FC<Props> = ({ activeFacilityId }) => {
   return (
     <main className="prime-home">
       <div className="grid-container queue-container-wide">
+        <h1 className="font-sans-lg">Conduct tests</h1>
```

Comment:
if you add a person to the queue I am seeing the same issue as above: "Heading levels should only increase by one"